docs/superpowers/plans/2026-04-30-acars-aircraft-tab.md

```
@@ -0,0 +1,1731 @@
+        // resolve to `None` so the viewer falls back to the bare
+        // 2-char code in the column display.
+        assert_eq!(lookup([0xFF, 0xFF]), None);
+        assert_eq!(lookup(*b"ZZ"), None);
+    }
+}
+```
+
+- [ ] **Step 2: Run tests to verify they fail**
+
+Run: `cargo test -p sdr-acars label::tests --features sdr-transcription/whisper-cpu`
+Expected: FAIL with `assertion `left == right` failed` for the positive lookups (the stub returns `None` for everything).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)